src/markdown-pages/install-with-kurl/adding-nodes.md

```diff
@@ -12,9 +12,48 @@ For instance, on an airgapped HA installation you would run `cat ./tasks.sh | su
 
 ## Standard Installations
 The install script will print the command that can be run on **worker** nodes to join them to your new cluster.
+
 ![add-nodes](/add-nodes.png)
 
 ## HA Installations
 For HA clusters, the install script will print out separate commands for joining **workers** and joining additional **master** nodes. 
 See [Highly Available K8s](/docs/install-with-kurl/#highly-available-k8s-ha) for HA install details.
+
 ![add-nodes-ha](/add-nodes-ha.png)
+
+## Resetting a Node
+
+It is possible to reset a node using the following script:
+
+*NOTE: The provided script is best effort and is not guaranteed to fully reset the node.*
+
+```bash
+curl -sSL https://kurl.sh/latest/tasks.sh | sudo bash -s reset
+```
+
+Or for airgap:
+
+```bash
+cat ./tasks.sh | sudo bash -s reset
+```
+
+## Rebooting a Node
+
+To safely reboot a node, use the following steps:
+
+*NOTE: In order to safely reboot a node it is required to have the [EKCO addon](/docs/add-ons/ekco) installed.*
+
+1. Run `/opt/ekco/shutdown.sh` on the node.
+1. Reboot the node
+
+## Removing a Node
+
+It is possible to remove a node from a multi-node cluster.
```

Comment:
```suggestion
It is possible to remove a node from a multi-node cluster.

When using the [rook add-on](/docs/add-ons/rook) extra precautions must be taken to avoid data loss.
Exec into the rook-ceph-operator or rook-ceph-tools pod in the rook-ceph namespace and run `ceph status`.
Ensure that any mons that are not in quorum are on the node to be removed.
```